.github/workflows/build.yml

```diff
@@ -298,3 +298,73 @@ jobs:
         if-no-files-found: error
         retention-days: 30
 
+  archlinux:
+    strategy:
+      matrix:
+        arch:
+          [
+            "linux/amd64 x86_64",
+            "linux/arm64/v8 aarch64",
+          ]
+    name: "Archlinux ${{ matrix.arch }}"
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+      with:
+        platforms: all
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v1
+      with:
+        version: latest
+    - name: Read info
+      id: tags
+      shell: bash
+      run: |
+        arch="${{ matrix.arch }}"
+        echo ::set-output name=PLATFORM::${arch%% *}
+        echo ::set-output name=ARCH::${arch##* }
+    - name: set environment variables
+      id: set_vars
+      run: ./scripts/ci-set-vars.sh
+      env:
+        REPOSITORY: ${{ github.event.repository.name }}
+    - name: Build ${{ matrix.arch }} release
+      shell: bash
+      run: |
+        arch="${{ matrix.arch }}"
+        PLATFORM=${arch%% *}
+        ARCH=${arch##* }
+        mkdir artifacts
+        docker buildx build --platform $PLATFORM \
+        --tag contour:$ARCH \
+        -f archlinux/Dockerfile \
+        --load \
+        .
+        docker create --name contour-$ARCH contour:$ARCH
+        docker cp contour-$ARCH:/app/contour.tar.zst contour-$ARCH.tar.zst
+        docker container rm contour-$ARCH
+    - name: "test: libcrispy"
+      shell: bash
+      run: |
+        arch="${{ matrix.arch }}"
+        PLATFORM=${arch%% *}
+        ARCH=${arch##* }
```

Comment:
Also here, and also in the task below. Maybe we can reduce the repetitive lines with a little helper script that these run tasks are simply calling? The script could be living in `/scripts/ci-SOME_NAME.sh`.

Reply:
So move this from below into a script and then use it for all other places so we have just one place where we do it? As these values should then be possible to access on every following task I would guess? I did not use github ci so much, that is why I ask.
```
        arch="${{ matrix.arch }}"
        echo ::set-output name=PLATFORM::${arch%% *}
        echo ::set-output name=ARCH::${arch##* }    
```

Reply:
When I am currently working on this change does it event work and to just to it for the two echo lines it worth it? As the `arch="${{ matrix.arch }}"` needs to be outside I would guess or am i wrong about it?

Reply:
ugh, the answer was yes. since I already merged now, I'll do that on next thursday/friday stream, I hope :)